usage/app/lib/UsageConfig.scala

```diff
@@ -62,9 +62,9 @@ class UsageConfig(override val configuration: Configuration) extends CommonConfi
 
   private val iamClient: AmazonIdentityManagement = withAWSCredentials(AmazonIdentityManagementClientBuilder.standard()).build()
 
-  val postfix: String = if (stage == "DEV") {
+  val postfix: String = if (isDev) {
     try {
-      iamClient.getUser().getUser().getUserName()
+      iamClient.getUser.getUser.getUserName
```

Comment:
At least some of these brackets (i.e the one denoting the side effect of an http request) are needed.